perfkitbenchmarker/traces/dstat.py

```diff
@@ -103,7 +103,7 @@ def _StopOnVm(self, vm):
     vm.RemoteCommand(cmd)
     try:
       vm.PullFile(self.output_directory, file_name)
-    except:
+    except Exception:
```

Comment:
Discussed offline. As a separate issue, we should address errors.VirtualMachine.RemoteExceptionError, which can also be raised in some situations.